README.md

```diff
@@ -42,6 +42,21 @@ server.start({
 });
 ```
 
+You can configure logging options as well via `logger`
+
+```js
+server.start({
+  logger: {
+    level: 'ALL',
+    appenders: [
+     { type: 'console' }
+   ]
+  },
+});
```

Comment:
Sorry, just a couple more changes, then it's ready to merge! 🎉 . Should probably add an example here of using a logger default values.